oda_api/api.py

```diff
@@ -60,13 +58,12 @@
 logger = logging.getLogger("oda_api.api")
 advice_logger = logging.getLogger("oda_api.advice")
 
-
 __all__ = ['Request', 'NoTraceBackWithLineNumber',
            'NoTraceBackWithLineNumber', 'RemoteException', 'DispatcherAPI']
 
 
 class Request(object):
-    def __init__(self,):
+    def __init__(self, ):
```

Comment:
why ","?

Reply:
I honestly don't know, this must be some results of a merging, or an automatic change applied by my IDE (and I missed it)